syntax/vlang.vim

```diff
@@ -209,6 +215,32 @@ endif
 hi def link    	vExtraType         Type
 hi def link    	vSpaceError        Error
 
+" Function calls and Fields are from: https://github.com/fatih/vim-go/blob/master/syntax/go.vim
+" Function calls;
+if v_highlight_function_calls
+  syn match vFunctionCall      /\w\+\ze(/ contains=goBuiltins,goDeclaration
+endif
+hi def link     vFunctionCall      Type
```

Comment:
Just a personal preference: could you change "Type" to "Special"?  I'm trying to reserve Type highlighting for types only. I'm going to change json.encode and json.decode to vBuiltins for the same reason.